src/registrar/assets/src/js/getgov-admin/helpers-portfolio-dynamic-fields.js

```diff
@@ -469,11 +470,36 @@ export function handlePortfolioSelection(
             if (requestedSuborganizationField) showElement(requestedSuborganizationField);
             if (suborganizationCity) showElement(suborganizationCity);
             if (suborganizationStateTerritory) showElement(suborganizationStateTerritory);
+            
+            // Handle rejectSuborganizationButtonFieldset (display of the clear requested suborg button).
+            // Basically, this button should only be visible when we have data for suborg, city, and state_territory.
+            // The function handleSuborgFieldsAndButtons() in domain-request-form.js handles doing this same logic
+            // but on field input for city, state_territory, and the suborg field.
+            // If it doesn't exist, don't do anything.
+            if (!rejectSuborganizationButtonFieldset){
+                console.warn("updateSuborganizationFieldsDisplay() => Could not update rejectSuborganizationButtonFieldset")
+                return;
```

Comment:
Why the early return here? I am having trouble following why the rejectSuborganizationButtonFieldset would not be present. Is it because it is only one the domain request page and not on domain and domain information change pages?

Can this and the next early return be removed, and the three if statements be combined into a nested if structure? Maybe with a comment to indicate that this block only applies to the domain request change view